code/datums/emergency_calls/upp_commando.dm

```diff
@@ -51,3 +51,53 @@
 
 	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(to_chat), H, SPAN_BOLD("Objectives: [objectives]")), 1 SECONDS)
 
+/datum/emergency_call/upp_commando/lowthreat
+	name = "UPP Commandos"
+	mob_max = 6
```

Comment:
Any of the vars that are the same here as the parent need to be removed. So like /datum/emergency_call/upp_commando already sets the mob_max to 6 and this inherits that so it does not need to be set here. Do a check for these dupes across the changes please.